stubs/docutils/docutils/readers/__init__.pyi

```diff
@@ -6,7 +6,7 @@ from docutils.io import Input
 from docutils.parsers import Parser
 from docutils.transforms import Transform
 
-_S = TypeVar("_S")
+_S = TypeVar("_S", bound=str | bytes)
```

Comment:
This is not correct because `_S` in `Input[_S]` means that the source from where we read is of type `_S`, but it can be a buffered object as well. It's too cumbersome to correctly type this one, so just leave it without any bound.

(See `docutils.io.FileInput` which uses `_S = IO[str]` or `docutils.io.NullInput` which uses `_S = Any`).

Reply:
is there an common interface they all share? or is literally *anything*?

Reply:
Literally nothing.